tests/base.py

```diff
@@ -201,6 +207,7 @@ def __init__(self, stack, visualizer=None, ax=None, tol=0.01, ext=".png",
 
         # Save other image comparison properties
         self.tol = tol
+        self.windows_tol = windows_tol
```

Comment:
My thought here is rather than adding a new class attribute, we could update self.tol using the logic you added on line 315, so change this to something like:

            if os.name == "nt" and windows_tol is not None:
            self.tol = windows_tol